### ISSUE TITLE
Making Webview localstorage persistant after app closed

### ISSUE THREAD

**Issue by rodrigograca31** [grevory/angular-local-storage#202]
Hi.

I'm using this module in an Cordova project and it works on the browser...
But on the Android App when I close the app it loses all the information stored on the localstorage....

I've noticed that the "[ngStorage](https://github.com/gsklee/ngStorage)" module does not do that... But I want to use this one....

Can you fix this problem or tell me what I need to change?

**Comment by grevory** [comment]
I haven't noticed or heard of this issue until now. I am not sure what is
causing it.

Can you create an Issue on Github so someone can look into it?

Thanks.


Greg

**Comment by rodrigograca31** [comment]
Hi @grevory this is an issue on Github....
You are replying from your email and got confused right...?

**Comment by grevory** [comment]
Yes. Thought this came in as a personal message. Not sure why. I clearly did not read it well.

**Comment by rodrigograca31** [comment]
I guess this is a problem with this particular module...
As I said before it works with another one...

And now I discovered that making it "by hand" also works: http://learn.ionicframework.com/formulas/localstorage/ (first code block)
